### DISCUSSION TITLE
qfield export project menu missing?

### DISCUSSION THREAD

**Discussion by sbec101** [opengisch/QField#2784] [Q&A]
Hello all, I had set up my qfield project working very well using dropbox, importing & exporting the projects and syncing that way. Now in the updated qfield , the app imports the project to a secured location on the device, which is fine, but... I need to get many people to compress this folder and send me the new data so I can sync the new data. I can't seem to find this menu anywhere, I have ready the instructions many times and it simply is not there or I am missing a step. Below is what I have been looking for. Can anyone please advise. Thanks in advance for your time ! 

![qfield export project](https://user-images.githubusercontent.com/102322524/164270476-1e0b6744-e733-45e1-af65-d1ef87479f0b.PNG)

**Comment by nirvn** [comment]
@sbec101 , are you running QField 2.0.15? If so, you will see the export actions by clicking on the open local file button in the QField welcome screen, and entering the imported projects folder. In there, you will have a menu button at the right end of every folder and subfolders in which the actions are found.

Note that it is not available in versions released prior to today, make sure you are uodated.

**Reply by sbec101** [reply]
I am / was running 2.0.14. I will give it a try. Although I did get it to work in qfield unstable. I am about to roll out 10 android tablets for data collection and this was a concern. Still trying to figure out how to get qfield cloud to connect to my amazon postgres/postgis data base. That's were I would like to end up. Thanks for the reply, I will give it a try in the updated version. Last question, is qfield still in beta sort off?

**Reply by nirvn** [reply]
@sbec101 , you'll definitely find those actions in 2.0.15 then. 

As for qfieldcloud, it sounds like a good fit for you reading your mention of deployment across 10 devices. It should handle postgis sources just fine.

**Reply by sbec101** [reply]
Sounds great but how to you do it, that information on how to use qfield cloud with my own existing amazon postgres/postgis server is not to be found , please advise, thx !
